src/main/java/seedu/duke/Duke.java

```diff
@@ -34,13 +34,23 @@ public static void run() {
             String command = new Parser().parseInput(userInput);
             switch (command) {
                 case "bye":
-                    System.out.println("Bye. Your flgihts have been recorded.\n Hope to see you again soon!");
+                    System.out.println("Bye. Your flights have been recorded.\n Hope to see you again soon!");
                     online = false;
                     break;
                 case "add":
                     flightList.addFlight(userInput);
                     System.out.println("Your flight has been added.\n" + "You have " + flightList.getSize() + " flights in your record");
                     break;
+                case "show all":
```

Comment:
the case command should be "show"

Variation of upcoming or all will be further implemented by the class.